app/controllers/host_aggregate_controller.rb

```diff
@@ -38,50 +31,6 @@ def new
     )
   end
 
-  def create
-    assert_privileges("host_aggregate_new")
```

Comment:
@MelsHyrule i don't think you should remove `assert_privileges("host_aggregate_new")` line, this is to verify if user has access to this page or not. check https://github.com/ManageIQ/manageiq-ui-classic/pull/7595/files for  a sample.